imaplugin/src/main/java/com/kaltura/playkit/plugins/ima/IMAConfig.java

```diff
@@ -49,6 +51,7 @@
     private static final String AD_COUNTDOWN_UIELEMENT_KEY    = "adCountDown";
     private static final String AD_LOAD_TIMEOUT_KEY           = "adLoadTimeOut";
     private static final String AD_MAX_REDIRECTS_KEY          = "adMaxRedirects";
+    private static final String AD_CONTENT_DURATION           = "adContentDuration";
```

Comment:
I think the name is a bit misleading, it is content duration but we prefix it with `ad`.
In general, why not get the real content duration from the player?
then - the flag we might want to expose is - `preloadPostRoll` as boolean and if it is true then don't set the duration at all.

Reply:
> I think the name is a bit misleading, it is content duration but we prefix it with `ad`. 

Done. 

> In general, why not get the real content duration from the player? 

We don't have content duration info at this time. Because content player is being prepared once the ad loads so we will not be able to pass this value in `adRequest`.

> then - the flag we might want to expose is - `preloadPostRoll` as boolean and if it is true then don't set the duration at all.

We thought of it but if we expose  `preloadPostRoll` then there is no point of exposing `setContentDuration` because app can set both and if they set contentDuration and  `preloadPostRoll` then there will be ambiguity.

We can either expose `preloadPostRoll` then we will loose `setContentDuration`